documentation/src/main/antora/modules/ROOT/pages/component-processor.adoc

```diff
@@ -123,7 +123,53 @@ public MyNewData map(final MyData data,
     return createNewData(data);
 }
 ----
+== Defining conditional outputs
 
+For each output, it can be defined show or not by using the annotation @ConditionalOutputFlows on the Processor and
```

Comment:
Processors @ElementListerner methods can declare several output flows. At design time, usually, all output flows are available, but, in certain cases, we want disabled some of them according to the configuration the user has already set. In that case, a service will be call that will return available output flows.